daemon/src/communication/ServerListener.cpp

```diff
@@ -120,12 +120,14 @@ void* executor(void* connector)
 	case OperationWithPathRequest::Create:
 		response = diskOps->create(
 			static_cast<OperationWithPathRequest*>(request)->getPath(),
-			static_cast<OperationWithPathRequest*>(request)->getMode());
+			static_cast<OperationWithPathRequest*>(request)->getMode(),
+			static_cast<OperationWithPathRequest*>(request)->getPid());
 		break;
 	case OperationWithPathRequest::Open:
 		response = diskOps->open(
 			static_cast<OperationWithPathRequest*>(request)->getPath(),
-			static_cast<OperationWithPathRequest*>(request)->getMode());
+			static_cast<OperationWithPathRequest*>(request)->getMode(),
+			static_cast<OperationWithPathRequest*>(request)->getPid());
```

Comment:
PID is already in conn (see getting FD few lines above)
please remove it from request

Reply:
Done